### PR TITLE
Implement events command for cgroup v1 stats

### PR DESCRIPTION
**Pull request by Furisto**

### CODE REVIEW

File: src/cgroups/v1/memory.rs [hunk 1]
```diff
@@ -86,7 +109,94 @@ impl Controller for Memory {
...
+        })
+    }
+
+    fn hierarchy_enabled(cgroup_path: &Path) -> Result<bool> {
+        let hierarchy_path = cgroup_path.join(MEMORY_USE_HIERARCHY);
+        let hierarchy = common::read_cgroup_file(hierarchy_path)?;
+        let enabled = match hierarchy.trim() {
+            "1" => true,
+            _ => false,
+        };
```
Comment (utam0k) [inline src/cgroups/v1/memory.rs:172]:
```suggestion
let enabled = matches!(hierarchy.trim(), "1");
```

File: src/cgroups/v1/blkio.rs [hunk 2]
```diff
@@ -33,46 +92,142 @@ impl Controller for Blkio {
...
 
         Ok(())
     }
+
+    fn get_throttling_policy_stats(cgroup_path: &Path) -> Result<BlkioStats> {
+        let mut stats = BlkioStats::default();
+
+        stats.service_bytes =
+            Self::parse_blkio_file(&cgroup_path.join(BLKIO_THROTTLE_IO_SERVICE_BYTES))?;
+        stats.serviced = Self::parse_blkio_file(&cgroup_path.join(BLKIO_THROTTLE_IO_SERVICED))?;
```
Comment (utam0k) [inline src/cgroups/v1/blkio.rs:145]: https://rust-lang.github.io/rust-clippy/master/index.html#field_reassign_with_default
```suggestion
        let stats = BlkioStats {
            service_bytes: Self::parse_blkio_file(
                &cgroup_path.join(BLKIO_THROTTLE_IO_SERVICE_BYTES),
            )?,
            serviced: Self::parse_blkio_file(&cgroup_path.join(BLKIO_THROTTLE_IO_SERVICED))?,
            ..Default::default()
        };
```

**Review by utam0k — CHANGES_REQUESTED** [review]
It seems to be mostly fine, but I made a few comments.

**Comment by Furisto** [comment]
@utam0k Updated with your suggestions.

**Review by utam0k — APPROVED** [review]
great!

**Pull request merged**